package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@matterlabs/zksync-contracts",
-  "version": "1.0.0-beta.3",
+  "version": "1.0.0-beta.4",
```

Comment:
The idea for the future: use version that matches the protocol version. Do not suggest doing it now though.